src/test/java/uk/gov/hmcts/reform/orgrolemapping/domain/service/PersistenceServiceTest.java

```diff
@@ -30,4 +31,17 @@ void getActorCacheEntity() throws IOException {
         Optional<RefreshJobEntity> response = sut.fetchRefreshJobById(1L);
         assertNotNull(response);
     }
+
+    @Test
+    void persistRefreshJobTest() {
+        RefreshJobEntity refreshEntity = RefreshJobEntity.builder()
+                .jobId(1L)
+                .roleCategory("role")
+                .jurisdiction("jurisdiction")
+                .status("NEW")
+                .created(LocalDateTime.now()).build();
```

Comment:
change this to ZonedDateTime.now() just for consistency. I don't think it matters in terms of the test. The actual object value should be Zoned anyway.